From d57839076e6116a7c5e684d6e8533f1757209672 Mon Sep 17 00:00:00 2001
From: Myles Watson <mylesgw@google.com>
Date: Wed, 3 Mar 2021 13:49:21 -0800
Subject: [PATCH 18/25] [Q_asb_2021-05] smp: Use SMP_TRACE_WARNING

Bug: 174886838
Test: pair an LE device
Tag: #security
Change-Id: I3c7e0b1dd877dfa29fa0812c6a5e65fcb1d70ea7
(cherry picked from commit 45678238713ba15cca8dd453b992caedf1d43ec5)
---
 stack/smp/smp_act.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/stack/smp/smp_act.cc b/stack/smp/smp_act.cc
index fc5098db63..ba7cbce8a3 100644
--- a/stack/smp/smp_act.cc
+++ b/stack/smp/smp_act.cc
@@ -692,7 +692,7 @@ void smp_process_pairing_public_key(tSMP_CB* p_cb, tSMP_INT_DATA* p_data) {
   if (!memcmp(p_cb->peer_publ_key.x, p_cb->loc_publ_key.x, BT_OCTET32_LEN) &&
       !memcmp(p_cb->peer_publ_key.y, p_cb->loc_publ_key.y, BT_OCTET32_LEN)) {
     android_errorWriteLog(0x534e4554, "174886838");
-    LOG_WARN("Remote and local public keys can't match");
+    SMP_TRACE_WARNING("Remote and local public keys can't match");
     tSMP_INT_DATA smp;
     smp.status = SMP_PAIR_AUTH_FAIL;
     smp_sm_event(p_cb, SMP_AUTH_CMPL_EVT, &smp);
-- 
2.35.1

